js/adapt-aboutUsSocialLinksView.js

```diff
@@ -5,38 +5,28 @@
  * Code was based on adapt-contrib-glossary and adapt-contrib-resources
  */
 
-define([
-    'core/js/adapt'
-], function(Adapt) {
-
-    var AboutUsSocialLinksView = Backbone.View.extend({
-
-        className: "aboutus__item",
-
-        attributes: {
-            role: 'listitem'
-        },
-
-        initialize: function() {
-            this.render();
-        },
-
-        render: function() {
-            var modelData = this.model;
-            var template = Handlebars.templates["aboutUsSocialLinks"];
-            this.$el.html(template(modelData));
-            _.defer(_.bind(function() {
-                this.postRender();
-            }, this));
-            return this;
-        },
-
-        postRender: function() {
-            this.listenTo(Adapt, 'drawer:openedItemView', this.remove);
-            this.listenTo(Adapt, 'drawer:triggerCustomView', this.remove);
-        }
-
-    });
-
-    return AboutUsSocialLinksView;
-});
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { templates } from 'core/js/reactHelpers';
+
+class AboutUsSocialLinksView extends Backbone.View{
+  initialize() {
+      this.render();
+  }
+
+  render() {
+    ReactDOM.render(<templates.aboutUsSocialLinks/>, this.el);
+
+    _.defer(_.bind(function() {
+        this.postRender();
+    }, this));
+    return this;
+  }
+
+  postRender() {
+    this.listenTo(Adapt, 'drawer:openedItemView', this.remove);
```

Comment:
Moving these listeners into the initialize will remove the need for a postRender